packages/flutter_tools/lib/src/web/file_generators/main_dart.dart

```diff
@@ -31,6 +46,8 @@ String generateMainDartFile(String appEntrypoint, {
     'Future<void> main() async {',
     '  await ui.webOnlyWarmupEngine(',
     '    runApp: () {',
+    if (_base64EncodedAssetManifest != null)
+      '      js.context["_flutter_base64EncodedAssetManifest"] = "$_base64EncodedAssetManifest";',
```

Comment:
Instead of base64 encoding this, you could instead inline this as a const list. That avoids the need for a base64 encoding.

I think that will work :)

Reply:
Done